fabfile.py

```diff
@@ -36,17 +36,17 @@ def build_static():
     local("./node_modules/.bin/gulp")
 
 
-def clear_memcached():
+def clear_redis_cache():
     try:
-        cache.init(config.MEMCACHED_SERVERS)
+        cache.init(config.REDIS_HOST, config.REDIS_PORT, config.REDIS_NAMESPACE)
         cache.flush_all()
```

Comment:
I am not running a Vagrant VM, so maybe I missed this. I'll try to look into it if I can.